### PR TITLE
[CLI] refactor the folder structure and components to be more generic

### PR DESCRIPTION
**Pull request by ludwig-pro**
## Description

- Rethink the folder structure to reduce ejs complexity

image

[Excalidraw link](https://excalidraw.com/#json=znvkpqES8umJeJVCPeIS-,l6nLarUCe1xvITaQ29fOtA)

## Motivation and Context

- be more flexible to modification, or new feature

## How Has This Been Tested?

- run each base / react-navigation / expo-router for each styling solution

### CODE REVIEW

File: cli/src/templates/packages/nativewind/components/EditScreenInfo.tsx.ejs [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+<% } %>
+
+
+export const EditScreenInfo = ({ path }: { path: string }) => {
+<% if (props.internalizationPackage?.name === "i18next") { %>
+  const { t } = useTranslation();
+  const getStarted = t('getStarted');
+  const changeCode = t('changeCode')
+<% } else { %>
+  const getStarted = "Open up the code for this screen:"
```
Comment (danstepanov) [inline cli/src/templates/packages/nativewind/components/EditScreenInfo.tsx.ejs:14]: change these to "title" and "description", something more generic

Comment (ludwig-pro) [reply cli/src/templates/packages/nativewind/components/EditScreenInfo.tsx.ejs]: done !

**Comment by ludwig-pro** [comment]
I also made the modification requested by @sammoore, only displaying the `<InternalizationExample />` component on one screen.

- main screen for stack
- modal screen for drawer
- modal screen for drawer + tabs

**Review by danstepanov — APPROVED** [review]
LGTM

**Pull request merged**